src/MainWindow.cpp

```diff
@@ -250,10 +250,13 @@ void MainWindow::OpenAbout(void)
 		"Raefaldhi Amartya Junior",
 		"Thomas Schmidt",
 		"waddlesplash",
+		"Tanausú Gómez (Spanish translation)",
 		NULL
 	};
 
 	abwin->AddCopyright(2009, "DarkWyrm");
+// Insertar la URL
```

Comment:
Generally comments should be in English as the coding lingua franca.
However, the code is totally obvious and doesn't need a comment. :)